ui/src/Annotation.tsx

```diff
@@ -14,6 +14,7 @@ interface AnnotationProps {
   id: string;
   className?: string;
   location: BoundingBox;
+  shouldHighlight?: boolean | false;
```

Comment:
Is this supposed to be `boolean | undefined` if `shouldHighlight` is optional?

Reply:
I mean, just `shouldHighlight?: boolean`?

Reply:
yes this should be shouldHighlight?: boolean?